### PR TITLE
[app] Add behavior to process blocks with gov txs sync

### PR DESCRIPTION
**Pull request by udpatil**
Tested locally with sei chain and gov txs

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -886,6 +895,9 @@ func (app *App) BuildDependencyDag(ctx sdk.Context, txs [][]byte) (*Dag, error)
 		}
 		msgs := tx.GetMsgs()
 		for messageIndex, msg := range msgs {
+			if isGovMessage(msg) {
+				return nil, ErrGovMsgInBlock
+			}
```
Comment (BrandonWeng) [inline app/app.go:900]: Hmm, it's not completely an error since this is expected right?

Comment (udpatil) [reply app/app.go:900]: yeah true, but this seemed like the best way to interrupt the dependency graph creation and have the sequential behavior. any idea on a better pattern to use?

File: scripts/old_initialize_local.sh [hunk 2]
```diff
@@ -1,43 +1,35 @@
...
-  -p 6831:6831/udp \
-  -p 6832:6832/udp \
-  -p 5778:5778 \
-  -p 16686:16686 \
-  -p 14250:14250 \
-  -p 14268:14268 \
-  -p 14269:14269 \
-  -p 9411:9411 \
-  jaegertracing/all-in-one:1.33
+#!/bin/bash
```
Comment (codchen) [inline scripts/old_initialize_local.sh]: are the changes below intended as part of the commit?

**Review by codchen — APPROVED** [review]

**Review by BrandonWeng — APPROVED** [review]
lgtm, reminder to undo changes for old init script

File: app/app.go (hunk 1 above)
Comment (BrandonWeng) [reply app/app.go:900]: Yeah, that's fair -- perhaps we could just leave a comment here noting why gov messages are handled this way

File: scripts/old_initialize_local.sh (hunk 2 above)
Comment (udpatil) [reply scripts/old_initialize_local.sh]: oops no

**Pull request merged**